| 2023-01-06 02:27:40,831 – [INFO]: Попытка подключения к PostgreSQL. Исполняемый файл – 'extract_from_pgsql.py': функция – 'connect_to_postgres'(25)
| 2023-01-06 02:27:40,913 – [INFO]: Успешное подключение к PostgreSQL. Исполняемый файл – 'extract_from_pgsql.py': функция – 'connect_to_postgres'(30)
| 2023-01-06 02:27:40,913 – [INFO]: Попытка подключения к Elasticsearch. Исполняемый файл – 'load_to_ES.py': функция – 'connect'(29)
| 2023-01-06 02:27:40,929 – [INFO]: Подключение к Elasticsearch: True. Исполняемый файл – 'load_to_ES.py': функция – 'connect'(31)
| 2023-01-06 02:27:40,951 – [DEBUG]: Файл /Users/madihanagatanov/middle_code/new_admin_panel_sprint_3/etl/state/state.json не может быть прочитан. Произошла ошибка: Expecting value: line 1 column 1 (char 0).. Исполняемый файл – 'storage.py': функция – 'retrieve_state'(45)
| 2023-01-06 02:27:41,402 – [DEBUG]: Файл /Users/madihanagatanov/middle_code/new_admin_panel_sprint_3/etl/state/state.json не может быть прочитан. Произошла ошибка: Expecting value: line 1 column 1 (char 0).. Исполняемый файл – 'storage.py': функция – 'retrieve_state'(45)
| 2023-01-06 02:27:41,403 – [INFO]: Успешно загружено 100 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:42,413 – [INFO]: Успешно загружено 200 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:43,528 – [INFO]: Успешно загружено 300 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:44,595 – [INFO]: Успешно загружено 400 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:45,678 – [INFO]: Успешно загружено 500 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:46,792 – [INFO]: Успешно загружено 600 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:47,816 – [INFO]: Успешно загружено 700 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:48,902 – [INFO]: Успешно загружено 800 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:49,967 – [INFO]: Успешно загружено 900 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:51,024 – [INFO]: Успешно загружено 999 объектов. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(27)
| 2023-01-06 02:27:51,024 – [ERROR]: Произошла ошибка при валидации фильмов из PostgreSQL. ['e72208b6-7181-49eb-9c25-2bc54629b21d']. Подробности: etl_status/state.json. Исполняемый файл – 'validators.py': функция – 'validate_film_works'(29)
| 2023-01-06 02:27:52,066 – [INFO]: Остановка процесса на 5 минут. Исполняемый файл – 'main.py': функция – 'main'(22)
